src/ui/Popover.tsx

```diff
@@ -0,0 +1,17 @@
+import * as PopoverPrimitive from '@radix-ui/react-popover';
+import { styled } from '@stitches/react';
+
+const StyledContent = styled(PopoverPrimitive.Content, {
+  padding: 0,
+  borderRadius: 8,
```

Comment:
should use value from theme tokens. `$3` should be equivalent to 8px
![image](https://user-images.githubusercontent.com/7143583/170187018-5fa96e18-05d2-4185-8022-1c41ce586cef.png)